Gemfile

```diff
@@ -11,6 +11,7 @@ gem "faraday"
 gem "faraday_middleware"
 gem "govuk_elements_rails"
 gem "govuk_template"
+gem "govuk_publishing_components", "~> 21.25.0"
```

Comment:
Is it a worry that Transport could be accidentally used at a later date as that font is bundled into the publishing components?

Reply:
I would say it's not a concern now for DGU right now, perhaps something to consider if we ever want to open up the gem to wider use though. 

I'm not sure on the plan for data.gov.uk, but it looks like [the font is allowed on gov.uk and *.service.gov.uk](https://designnotes.blog.gov.uk/2015/03/11/can-i-use-the-gov-uk-fonts/), so it's possible that could be extended to DGU if we look at bringing it in line visually with gov.uk in the future.